vllm/model_executor/models/mixtral.py

```diff
@@ -275,10 +303,17 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
             org_num_embeddings=config.vocab_size,
         )
 
+        self.enable_eplb = parallel_config.enable_eplb
+        self.num_redundant_experts = parallel_config.num_redundant_experts
```

Comment:
```suggestion
        self.num_redundant_experts = parallel_config.eplb_config.num_redundant_experts
```

Same as above. Note: `enable_eplb` is still inside `parallel_config` so no need to change